app/src/debug/AndroidManifest.xml

```diff
@@ -9,7 +9,11 @@
     <uses-permission android:name="android.permission.WAKE_LOCK"/>
 
     <!-- Allows for storing and retrieving screenshots -->
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission
+        android:name="android.permission.WRITE_EXTERNAL_STORAGE"
+        android:maxSdkVersion="28"
```

Comment:
@Amejia481 would you know if this would have an effect for SDK 29+ in downloads? Do we handle that already?

I'm wondering if settings this attribute means that we can't remove it and publish new updates if something isn't working.

(Same comment for the `main` manifest.

Reply:
As we are already using the scoped storage, we shouldn't have issues,
as the permissions doesn't have any effect on devices with [SDK 29+](https://developer.android.com/about/versions/11/privacy/storage#permissions-target-11).

We recently did something similar on [feature-downloads ](https://github.com/mozilla-mobile/android-components/pull/10686/). To be extra careful we can do a round of test just to be sure.

Reply:
I tested with SKD 24, 28, 29 and 30 it worked as expected :)

Reply:
Great